include/openmc/wmp.h

```diff
@@ -0,0 +1,75 @@
+#ifndef OPENMC_WMP_H
+#define OPENMC_WMP_H
+
+#include "hdf5.h"
+#include "xtensor/xtensor.hpp"
+
+#include <complex>
+#include <string>
+#include <tuple>
+
+namespace openmc {
+
+//========================================================================
+// Constants
+//========================================================================
+
+// Constants that determine which value to access
+constexpr int MP_EA {0}; // Pole
+constexpr int MP_RS {1}; // Residue scattering
+constexpr int MP_RA {2}; // Residue absorption
+constexpr int MP_RF {3}; // Residue fission
+
+// Polynomial fit indices
+constexpr int FIT_S {0}; // Scattering
+constexpr int FIT_A {1}; // Absorption
+constexpr int FIT_F {2}; // Fission
+
+//========================================================================
+// Windowed multipole data
+//========================================================================
+
+class WindowedMultipole {
+public:
+  // Types, aliases
+  using cdouble = std::complex<double>;
```

Comment:
This alias is only used in one place so I think we may as well just stick with the long name.